### ISSUE TITLE
Disable template loading during update

### ISSUE THREAD

**Issue by luckymike** [sparkleformation/sfn-parameters#11]
I'd like to have the option of defining all my stacks using stack files, with the template defined, but maintain the ability to do parameters-only updates.

Is there a way to disable template loading (i.e. preserve the default `sfn update` parameters-only behavior) when a template is defined in a stack file?

**Comment by chrisroberts** [comment]
I don't believe there is a way to do this currently without removing the template name setting in the file. I think the best approach may be to add a new configuration item to enable/disable template loading. I'll play around with that and see if it is a workable solution.
